generators/chipyard/src/main/scala/config/TracegenConfigs.scala

```diff
@@ -42,8 +50,10 @@ class NonBlockingTraceGenL2Config extends Config(
   new freechips.rocketchip.groundtest.GroundTestBaseConfig)
 
 class NonBlockingTraceGenL2RingConfig extends Config(
-  new chipyard.iobinders.WithBlackBoxSimMem ++
-  new chipyard.iobinders.WithTraceGenSuccessBinder ++
+  new chipyard.harness.WithBlackBoxSimMem ++
+  new chipyard.harness.WithTraceGenSuccess ++
+  new chipyard.iobinders.WithAXI4MemPunchthrough ++
+  new chipyard.iobinders.WithTraceGenSuccessPunchthrough ++
```

Comment:
You might consider having a `TraceGenAbstractConfig` to reduce the duplication here.